rosdep/python.yaml

```diff
@@ -497,6 +501,8 @@ python-catkin-pkg:
     pip:
       packages: [catkin-pkg]
   ubuntu:
+    artful: [python-catkin-pkg]
+    bionic: [python-catkin-pkg]
```

Comment:
should these have a `_python3` equivalent like it was done for the kinetic ? (same for mock and nose below)

Reply:
Since that depends on how we want Python 3 testing to happen I didn't add them at this time.